src/queue.rs

```diff
@@ -566,6 +566,76 @@ impl<T: Task> Queue<T> {
         Ok(id)
     }
 
+    #[instrument(
+        name = "enqueue",
+        skip(self, executor, task, inputs),
+        fields(queue.name = self.name, task.id = tracing::field::Empty),
+        err
+    )]
+    pub async fn enqueue_multi<'a, E>(
+        &self,
+        executor: E,
+        task: &T,
+        inputs: &[T::Input],
+    ) -> Result<Vec<TaskId>>
+    where
+        E: PgExecutor<'a>,
+    {
+        let mut ids = Vec::with_capacity(inputs.len());
+        let mut input_values = Vec::with_capacity(inputs.len());
+        let mut delays = Vec::with_capacity(inputs.len());
+
+        for input in inputs {
+            ids.push(TaskId::new());
+            input_values.push(serde_json::to_value(input)?);
+            delays.push(StdDuration::try_from(task.delay())?);
+        }
+
+        let timeout = task.timeout();
+        let heartbeat = task.heartbeat();
+        let ttl = task.ttl();
+        let retry_policy = task.retry_policy();
+        let concurrency_key = task.concurrency_key();
+        let priority = task.priority();
+
+        // what use here? dont trace?
```

Comment:
@maxcountryman should here b trace?

Reply:
Maybe instead of the task ID we can record the number of tasks in the batch?

Reply:
sure